context/brand-context/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # History
 
+## 31.1.0 (2023-01-09)
+    * BREAKING:
```

Comment:
If this is breaking, should it not be version 32?

Reply:
@alexkilgour True, I forgot to sync that when I made it breaking.